src/main/java/nextstep/ladder/domain/Ladder.java

```diff
@@ -0,0 +1,60 @@
+package nextstep.ladder.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Ladder {
+    private List<Integer[]> list;
```

Comment:
```suggestion
    private List<Line> lines;
```
Line 객체를 분리하고 적절한 책임을 부여 함으로서 
해당 객체에 의미를 부여해보는게 어떨까요? 😄